operators/cmd/tenant-operator/main.go

```diff
@@ -89,6 +90,7 @@ func main() {
 	flag.StringVar(&ncTnOpPsw, "nc-tenant-operator-psw", "", "The password of the acting account for nextcloud.")
 	flag.IntVar(&maxConcurrentReconciles, "max-concurrent-reconciles", 1, "The maximum number of concurrent Reconciles which can be run")
 	flag.StringVar(&webhookBypassGroups, "webhook-bypass-groups", "system:masters", "The list of groups which can skip webhooks checks, comma separated values")
+	flag.BoolVar(&enableWH, "enable-webhooks", true, "Enable webhooks server")
```

Comment:
```suggestion
	enableWH := flag.Bool("enable-webhooks", true, "Enable webhooks server")
```

Nit. It avoid to define the variable in advance.
Cloud also set it to false in the makefile, for local execution.